### DISCUSSION TITLE
Consider symbols when sorting by name

### DISCUSSION THREAD

**Discussion by AZeed18** [community/community#40552] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

I usually use symbols or emojis to pin something to top when sorting by name, but symbols and emojis are ignored when sorting lists by name

**Comment by ossanna16** [comment]
Hi there @AZeed18 and welcome to our community! Thank you for the feedback!

**Reply by Yowyaz** [reply]
Data fraud

**Reply by AZeed18** [reply]
> Data fraud

What do you mean?

**Comment by Vovanchik25071982** [comment]
> ### Select Topic Area
> 
> Product Feedback
> 
> ### Body
> 
> I usually use symbols or emojis to pin something to top when sorting by name, but symbols and emojis are ignored when sorting lists by name

**Comment by Chasefundzbtc** [comment]
Hi

**Comment by Chasefundzbtc** [comment]
Select Topic Area

Product Feedback

Body

I usually use symbols or emojis to pin something to top when sorting by name, but symbols and emojis are ignored when sorting lists by name
